docs/README.md

```diff
@@ -0,0 +1,114 @@
+@asyncapi/diff / [Modules](modules.md)
```

Comment:
Any reason why we are duplicating the README file inside `docs/README` as well?

Reply:
the README inside `docs` is automatically generated. It usually copies the existing `README.md` present in the root. Let me check if this is avoidable.

Reply:
Is this the bug you were talking about?